### PR TITLE
NET7 and small cleanup

### PR DESCRIPTION
**Pull request by sungam3r**
If some changes seem superfluous, I will revert them. In fact, there are not many changes at all and they are cosmetic.

### CODE REVIEW

**Comment by paulomorgado** [comment]
There are quite a few style and source text changes.

Please, back them up with a .editorconfig file.

**Comment by sungam3r** [comment]
What exactly would you like me to revert? Switch expressions?

File: src/Microsoft.IO.RecyclableMemoryStream.csproj [hunk 1]
```diff
@@ -50,11 +51,11 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
     <PackageReference Include="System.Memory" Version="4.5.5" Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'netstandard2.0'" />
   </ItemGroup>
-  
+
   <ItemGroup>
-    <None Include="..\README.md" Pack="true" PackagePath="\" />
+    <None Include="..\README.md" Pack="true" PackagePath="\" Visible="false" />
```
Comment (benmwatson) [inline src/Microsoft.IO.RecyclableMemoryStream.csproj:56]: Why make this invisible?

**Comment by benmwatson** [comment]
> What exactly would you like me to revert? Switch expressions? Using statements? Both?

He's not saying to revert, just to add an .editorconfig to enforce these style decisions automatically. I am neutral on the specific style used, but I think the .editorconfig is a good idea to enforce style decisions going forward.

Overall, no issues with PR. Just a minor question about why you want to hide README.md in the project. I appreciate having it visible in VS.

File: src/Microsoft.IO.RecyclableMemoryStream.csproj (hunk 1 above)
Comment (sungam3r) [reply src/Microsoft.IO.RecyclableMemoryStream.csproj:56]: Already visible in solution items.

**Comment by sungam3r** [comment]
> Please, back them up with a .editorconfig file.

I misread it. Done.

**Pull request merged**